### PR TITLE
TravisCI integration

### PR DESCRIPTION
**Pull request by Seanny123**
First draft of attempting to integrate to TravisCI.

Todo:
- [ ] Fix failing tests
- [ ] Reduce code duplications by importing most of `test_examples.py` from `nengo`

### CODE REVIEW

**Comment by tbekolay** [comment]
TravisCI is enabled now, it should build the next time you push.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+# Run the tests
+script:
+  - mkdir -p "$HOME/.ipython/profile_default"
+  - "echo 'c.HistoryAccessor.enabled = False\n' > $HOME/.ipython/profile_default/ipython_config.py"
+  - if [[ "$STATIC" == "false" ]]; then
+      python -c "import numpy; numpy.show_config()";
+      python setup.py -q develop;
+      if [[ "$COVERAGE" == "false" ]]; then
+        py.test nengo -n 2 -v --duration 20;
```
Comment (Seanny123) [inline .travis.yml]: Note: should remove test coverage and figure out actual command to run tests.

**Comment by Seanny123** [comment]
@jgosmann I messed up rebasing again, but aside from that, I'm a bit confused about how to resolve the remaining pylint errors. Would you mind taking a look?

**Comment by jgosmann** [comment]
Well those should be solved by pretty much what it is recommending:
1. Add the `r` prefix to the (doc) strings it is complaining about in `nengo_extras.learning_rules`.
2. Remove the unused import of `StringIO` in `nengo_extras.utils`.

**Comment by Seanny123** [comment]
> Well those should be solved by pretty much what it is recommending

I literally have no idea why I was confused last night.

**Comment by tbekolay** [comment]
This was superseded by the now merged #68.

**Pull request closed without merging**